app/package.json

```diff
@@ -34,35 +34,35 @@
     "@mui/material": "~5.15.10",
     "@mui/styled-engine": "5.15.11",
     "@mui/system": "~5.15.12",
-    "@mui/x-date-pickers": "^7.12.0",
+    "@mui/x-date-pickers": "^7.13.0",
     "@react-keycloak/web": "^3.4.0",
     "@turf/bbox": "~6.5.0",
     "@turf/boolean-equal": "~6.5.0",
     "@turf/centroid": "~6.5.0",
     "@turf/turf": "^6.5.0",
     "@zip.js/zip.js": "^2.7.48",
-    "axios": "^1.7.3",
+    "axios": "^1.7.4",
     "buffer": "^6.0.3",
     "clsx": "~2.1.0",
-    "dayjs": "^1.11.12",
+    "dayjs": "^1.11.13",
     "express": "~4.18.3",
     "formik": "~2.4.5",
     "keycloak-js": "~23.0.7",
     "lodash-es": "~4.17.21",
     "maplibre-gl": "^4.5.0",
     "oidc-client-ts": "^3.0.1",
     "qs": "~6.11.2",
-    "react": "~18.2.0",
```

Comment:
Did you mean to update react version here?

Reply:
yes, and as we discussed we can lock that version from now on and if there's a good reason to upgrade we can discuss it and decide.